source/includes/_01-your-first-backend-in-10-minutes.md

```diff
@@ -0,0 +1,204 @@
+# Building your first serverless backend in 10 minutes 
+
+Building your serverless application backend have never been this easy. In this section, we will go through all the necessary steps to have your backend up and running in a few minutes.
+
+If you follow the steps below, you will be able to create the application backend for a blog.
+
+The steps to follow will be:
+- [Create project](#1-create-the-project)
+- [First command](#2-first-command)
+- [First event](#3-first-event)
+- [First entity](#4-first-entity)
+- [First read model](#5-first-read-model)
+- [Deployment](#6-deployment)
+- [Testing](#7-testing)
+    - [Creating posts](#71-creating-posts)
+    - [Retrieving all posts](#72-retrieving-all-posts)
+    - [Retrieving specific post](#73-retrieving-specific-post)
+- [Removing stack](#8-removing-stack)
+- [Further improvements](#9-further-improvements)
+- [Conclusion](#10-conclusion)
+
+## 1. Create project
+First of all, we need to create a base Booster project
+
+`boost new:project <project-name>`
+
+You will need to answer a few questions in order to configure the project. One of these steps will require that a provider is selected, we recommend selecting AWS. 
+
+Once the project has been successfully created, you will need to move to the new directory. Ideally, you would also open the project in your most preferred IDE, e.g. Visual Studio Code.
+
+## 2. First command
+We will now define our first command, which will allow us to create Posts.
+
+`boost new:command CreatePost --fields postId:UUID title:string content:string author:string`
```

Comment:
Note that we use `ID` in all capitals. It shouldn't matter, but just to be consistent with the rest of the project.

```suggestion
`boost new:command CreatePost --fields postID:UUID title:string content:string author:string`
```